content/collections/create.en.md

```diff
@@ -193,7 +193,7 @@ Before proceeding with deployment, ensure that the server meets the following re
   - Add the following line at the end of this section:
   
     ```shell
-    <user>  ALL=(ALL) NOPASSWD:ALL
+    <username>  ALL=(ALL:ALL) NOPASSWD:ALL
```

Comment:
Why do you need to allow the user to specify both the target user and group when using `sudo`?